### ISSUE TITLE
[Feature Request]: "Deep profile" selection at startup and renaming.

### ISSUE THREAD

**Issue by autonum** [FreeTubeApp/FreeTube#4084]
### Guidelines

- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.
- [X] This issue contains only one feature request.

### Problem Description

Based on my own experience with FreeTube and other applications, I think it would come very handy for certain users to add a profile selection at startup. Just like Thunderbird and Firefox allow for.
Especially since the «Playlists» feature has not really been implemented and acts only as a «Favorite videos» list ;
and, even though it would have been, being able to have completely different experiences, with different settings, etc', and clearly distinguish contexts could prove very useful and not cosmetic.

(As now, I managed to simulate this behavior : I added a few new FT.desktop that open FT with a specific folder in .conf. Which is exactly what the requested feature would do, but without having multiple FT icons in the dock.
Also, with this option, the name of the application appearing in the top bar always show the same .conf folder name (for example, «FreeTube-music»), regardless which one the application is running on. Strange.

### Proposed Solution

Add an option to be able to select a profile at startup.
Profiles would have their own configuration directory (.conf/FreeTube/profileX or .conf/FreeTube-profileX).
This way, one could have, for example, one specific profile for «Music». It would have different subscriptions related to music, those subscriptions could be grouped under several «Theme»(?) (now called «Profile» in FreeTube), and all the present features of FT.
Then one could have another profile for «Woodwork». Which could also have different subscriptions, grouped under different «Themes», etc'.

So, basically, it would add a superior layer of profile distinction.
Like Thunderbird allows not only to have multiple accounts (mail, RSS, etc') but to have multiple profiles which all have their own set of multiple accounts.

The only change would be this (optional) profile selection (and manager) at startup.

### Alternatives Considered

A very complete «Playlists» feature.
Multiple playlists, with auto-play and all, and with "playlist profiles" to be able to group different playlists under a same topic. Like folders and sub-folders containing playlists.

### Issue Labels

ease of use improvement, improvement to existing feature, new feature

**Comment by absidue** [comment]
Sounds like a duplicate of #1996

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Yup
